dts/bindings/pinctrl/espressif,esp32-pinmux.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2019 Mohamed ElShahawi
+# SPDX-License-Identifier: Apache-2.0
+title: ESP32 PINMUX
+
+description: >
+    This binding gives a base representation of the ESP32 PINMUX
+
+compatible: "espressif,esp32-pinmux"
+
+include: base.yaml
+
+properties:
+    reg:
+      required: true
+
+pinmux-cells:
```

Comment:
I wondering if we should drop this part for now.  Until we have actual pins defined in DTS and a means to generate that information.

Reply:
i can see the same property in all yaml files, but not being used in dts except `nxp_xxxxxx.dtsi`
So, i'm not sure what is the correct action. is there a problem of let this property here or should i remove it and cleanup the other yaml files ?

Reply:
For now lets remove it.  We can add it when it needed.